proposals/lip-0061.md

```diff
@@ -570,7 +573,9 @@ def verifyAggregateCommit(blockHeader: BlockHeader)-> bool:
     # Note that the BFT parameters at height maxHeightCertified+1 are already authenticated
     # by the previous certificate.
     heightNextBFTParameters = getNextHeightBFTParameters(getBFTHeights().maxHeightCertified+1)
-    if previous function call returns valid height and aggregateCommit.height > heightNextBFTParameters-1:
+    if previous function call returns valid height
+     and heightNextBFTParameters > MIN_CERTIFICATE_HEIGHT
```

Comment:
So with the additional condition we require that a certificate is at the earliest required for the last block of the round that starts with height `MIN_CERTIFICATE_HEIGHT`.

I think what we additionally should check is that we do not have any certificate below height `MIN_CERTIFICATE_HEIGHT` so we are guaranteed to have a chain of trust.

Reply:
Indeed, I had this check before and it was erroneously removed. Check now.